README.md

```diff
@@ -18,7 +18,7 @@ Open-source maintainers are always looking to get more people involved, but new
 
 You're welcome to add a new project in Good First Issue, and we encourage all projects &mdash; old and new, big and small.
 
-Follow these simple steps:
+### <b>Follow these simple steps:</b>
```

Comment:
Please remove the `###` declaration, since this will convert the line into an `<h3>`.

Reply:
Sure sir, I will remove "###".

Reply:
This is my first contribution and as per your suggestion i had removed "###".